api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -870,12 +869,6 @@ public ResponseEntity<RecentWorkspaceResponse> updateRecentWorkspaces(
     return ResponseEntity.ok(recentWorkspaceResponse);
   }
 
-  @Override
-  public ResponseEntity<WorkspaceResourceResponse> getWorkspaceResources(
-      String ns, String id, WorkspaceResourcesRequest wrr) {
-    return getWorkspaceResourcesImpl(ns, id, wrr.getTypesToFetch());
-  }
-
   @Override
   public ResponseEntity<WorkspaceResourceResponse> getWorkspaceResourcesV2(
```

Comment:
Should this be renamed to getworkspaceResources?

Reply:
SO in case this PR goes in removing getWorkspaceResponse, wait for release and then rename?

Reply:
yeah that's what I'd prefer.   I might be OK with leaving this "v2" too

Reply:
my concern is that v2 will loose context/meaning in near future so better to loose it